### ISSUE TITLE
How to pass options within Emacs?

### ISSUE THREAD

**Issue by realjsk** [minoki/cluttex#16]
How to pass options (e.g., --shell-escape) within Emacs?

**Comment by minoki** [comment]
I don't get the meaning of "within Emacs". Are you using some Emacs mode to invoke cluttex? Or, do you want to specify options with some form of magic comment?

**Comment by realjsk** [comment]
With auctex activated in Emacs, I can compile tex files with C-c C-c. This command gives me a set of options to run (e.g., LaTeX, ClutTex, BibTeX, etc.) I can choose ClutTeX from that list of options to compile the file using ClutTeX, but I don't know how to pass ClutTeX options (e.g., --shell-escape).

**Comment by minoki** [comment]
AFAIK, AUCTeX doesn't provide ClutTeX as an option by default. Are you using something like [tsuu32/auctex-cluttex](https://github.com/tsuu32/auctex-cluttex)? There's a similar issue (with no response): https://github.com/tsuu32/auctex-cluttex/issues/10

Unfortunately I am not familiar with customization of Emacs, so there is not much I can do to help you.

**Comment by wail-uottawa** [comment]
Yes, I'm using [tsuu32/auctex-cluttex](https://github.com/tsuu32/auctex-cluttex). It's me who left that issue (https://github.com/tsuu32/auctex-cluttex/issues/10), but unfortunately, I received no response. I understand that my question may be beyond your expertise. Thank you for being responsive.

**Comment by realjsk** [comment]
Yes, I'm using [tsuu32/auctex-cluttex](https://github.com/tsuu32/auctex-cluttex). It's me who left that issue (https://github.com/tsuu32/auctex-cluttex/issues/10), but unfortunately, I received no response. I understand that my question may be beyond your expertise. Thank you for being responsive.

**Comment by ecofighter** [comment]
Hello,
I wrote some advice about this in tsuu32/auctex-cluttex#10. It might also help you.

**Comment by realjsk** [comment]
Thank you!
